ckan-2.10/base/setup/supervisord.conf

```diff
@@ -1,7 +1,7 @@
 [unix_http_server]
 file = /tmp/supervisor.sock
 chmod = 0777
```

Comment:
If we're still using supervisor this doesn't look like a safe set of file permissions. If we aren't running supervisor we could delete this file.

Reply:
Yes the plan is still to remove Supervisor. I see the "remove supervisor PR": https://github.com/ckan/ckan-docker-base/pull/77 has been approved but not merged...Were you waiting for me to merge? I can...however I'd better bring in the recent changes to master to make sure I don't mess things up

Reply:
Yes I approved but didn't merge in case you there was some reason to wait for any other updates. We don't have releases or a stable branch for this repo yet. Please go ahead and merge if it makes sense